site/public/js/grade-inquiry.js

```diff
@@ -1,5 +1,23 @@
 /* global buildCourseUrl, WebSocketClient */
-/* exported initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
+/* exported loadDraft, initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
+
+function loadDraft() {
+    // LoadDraft function displays the saved draft text on grade inquiry box
+    let draftContent = localStorage.getItem('draftContent');
```

Comment:
This key isn't very unique.  Won't grade inquiries on different gradeables conflict with each other?